### PR TITLE
[stable/pomerium] Setting Ingress Hosts from policy

### PR DESCRIPTION
**Pull request by mohsen0**
The kubernetes-sigs/external-dns create DNS records based on the Hostnames in Ingress objects.
Setting the hostnames explicitly in the Ingress object and not using a wildcard can help to take advantage of this functionality

https://github.com/pomerium/pomerium-helm/pull/32

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/pomerium/templates/ingress.yaml [hunk 1]
```diff
@@ -16,29 +16,49 @@ spec:
   tls:
     - secretName: {{ default .Values.ingress.secretName .Values.ingress.secret.name}}
       hosts:
-        - {{ printf "*.%s" .Values.config.rootDomain | quote }}
+        - {{ printf "authorize.%s" .Values.config.rootDomain | quote }}
+        - {{ printf "authenticate.%s" .Values.config.rootDomain | quote }}
+      {{- range .Values.config.policy }}
```
Comment (travisgroth) [inline stable/pomerium/templates/ingress.yaml:22]: Can you make this mutually exclusive with the explicit ingress.hosts list?

**Review by travisgroth — CHANGES_REQUESTED** [review]
One small tweak and then LGTM.

File: stable/pomerium/templates/ingress.yaml (hunk 1 above)
Comment (mohsen0) [reply stable/pomerium/templates/ingress.yaml:22]: Done

Comment (travisgroth) [reply stable/pomerium/templates/ingress.yaml:22]: 👍

**Comment by travisgroth** [comment]
@mohsen0 looks good but you'll need to either sign that last commit or squash it first to appease the bots.  Thanks again!

**Pull request merged**